gateways/rocketgate/request_builders.go

```diff
@@ -47,7 +47,7 @@ func buildAuthRequest(merchantID string, merchantPassword string, merchantAccoun
 	}
 
 	// Ignore CVV and AVS check
-	gatewayRequest.Set(request.CVV2_CHECK, "IGNORE")
+	gatewayRequest.Set(request.CVV2_CHECK, "NO")
 	gatewayRequest.Set(request.AVS_CHECK, "IGNORE")
 
 	return gatewayRequest
```

Comment:
I cannot comment on those lines, but for `buildCaptureRequest` and `buildRefundRequest`, you also need to change to 
`gatewayRequest.Set(request.AMOUNT, sleet.AmountToDecimalString(captureRequest.Amount))`
`gatewayRequest.Set(request.AMOUNT, sleet.AmountToDecimalString(refundRequest.Amount))`

Reply:
haha realized that when the refund and capture tests were failing. I fixed that